geniza/corpus/tests/test_corpus_admin.py

```diff
@@ -178,7 +178,7 @@ def test_save_model(self, db):
     def test_get_search_results(self, document, join):
         # index fixture data in solr
         Document.index_items([document, join])
-        time.sleep(1)
+        time.sleep(2)
```

Comment:
low priority, but is there a way to poll solr in order to know when the operation finishes? or is it more like an eventually-consistent database, where the request will always complete but the database operation is never guaranteed to have happened at any particular time (just "eventually")

Reply:
I don't know! The tests started failing in CI and it seemed like this was the culprit — I think it was after the indexing added more data.  However... I remember now we added a solr indexing config for tests in one of the other projects, maybe we can add that. (I don't think we've added it here)